nevergrad/functions/test_base.py

```diff
@@ -6,7 +6,7 @@
 import pickle
```

Comment:
changes to this files are unrelated, just a convention update since I stumbled on it in the process